.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.7.6
```

Comment:
I think it's better to ship a version of Ruby with this gem that we as developers use, so that we know we are using the same version. Much easier to debug things when you know everyone is using the same version...